app/Locale/bg_BG/translations.php

```diff
@@ -134,10 +134,10 @@
     'User removed successfully.' => 'Потребителят е премахнат успешно.',
     'Unable to remove this user.' => 'Този потребител не може да бъде премахнат.',
     'Board updated successfully.' => 'Дъската е актуализиран успешно.',
-    'Ready' => 'Готово',
-    'Backlog' => 'Списък на чакащите задачи' ,
+    'Ready' => 'В готовност',
+    'Backlog' => 'Изчакващи' ,

```

Comment:
You have a whitespace betwwen the apostrophe and the comma, this may cause errors.

Reply:
Sorry, I've fixed it.